cockroachcloud/connect-to-a-serverless-cluster.md

```diff
@@ -62,7 +62,7 @@ To connect to your cluster with the [built-in SQL client](../{{site.versions["st
 
     {% include cockroachcloud/download-the-binary.md %}
 
-1. In your terminal, run the second command from the dialog to create a new `certs` directory on your local machine and download the CA certificate to that directory:
+1. In your terminal, run the second command from the dialog to download the CA certificate to the default PostgreSQL certificate directory:
```

Comment:
Looking at the current **Connect** dialog, it seems like the second command was removed?